java/ql/src/Security/CWE/CWE-074/JndiInjection.ql

```diff
@@ -11,8 +11,7 @@
  */
 
 import java
-import semmle.code.java.dataflow.FlowSources
-import JndiInjectionLib
+import semmle.code.java.security.JndiInjectionQuery
```

Comment:
This comment is for line 3 (query description)

Suggesting the following:

Performing a JNDI lookup with a user-controlled name can lead to the download of an untrusted object and to execution of arbitrary code.

Reply:
Thank you for the review @mchammer01! I applied your suggestion in 29490e5.